ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/order-confirmation-email-template.html

```diff
@@ -7,14 +7,14 @@
 
 Your Order ID is ${orderId}.
 
-Total Amount:${grandTotal}
+Total Amount: $ ${grandTotal}
 
 Shipping Address: ${address}
 
 
 Manage Your Orders At: http://localhost:4502/content/hclecomm/us/en/orders.html
 
-Click on this link to continue shopping with us!! 
+Click on this link to continue shopping with us!!
 http://localhost:4502/content/hclecomm/us/en/home.html
```

Comment:
@anchal-a : can we remove the absolute URL path & make it relative.

Reply:
Yes Rishabh, This bug is under process. It needs to be done in all the templates. So will fix it later in all the templates.